src/components/vm/hostdevs/hostDevAdd.jsx

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
Commit message "Enable an addition of host devices to VM" sounds strange. How about "Add host devices to VM"?